pyproject.toml

```diff
@@ -31,7 +31,7 @@ dynamic = ["version", "readme"]
 dependencies = [
     "aiohttp >= 3.9.2, < 4.0.0",
     "aioitertools >= 0.5.1, < 1.0.0",
-    "botocore >= 1.40.46, < 1.40.50", # NOTE: When updating, always keep `project.optional-dependencies` aligned
+    "botocore >= 1.40.46, < 1.40.61", # NOTE: When updating, always keep `project.optional-dependencies` aligned
```

Comment:
I believe 1.40.61 was recently shipped and does not include any breaking changes for us. Could we relax the dependency specification further, i.e. to `< 1.40.62`?

Reply:
Now we only need to update the lock file one more time...